ec2/tags.go

```diff
@@ -0,0 +1,44 @@
+package ec2
+
+import (
+	"context"
+
+	"github.com/YaleSpinup/apierror"
+	"github.com/YaleSpinup/ec2-api/common"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	log "github.com/sirupsen/logrus"
+)
+
+func (e *Ec2) UpdateTags(ctx context.Context, id string, rawTags map[string]string) error {
+	if id == "" || len(rawTags) == 0 {
+		return apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+	resources := e.getResourcesById(id)
+	tags := toEc2Tags(rawTags)
```

Comment:
When a function is small and not reused, as in this case, it's better to just put the for loop here.
Too many functions make the code harder to read.

Reply:
I was checking the ruby code and noticed that when we want to update tags for instances, we may have more than one resource in that case we can use this function.